plugins/catalog-backend/src/catalog/types.ts

```diff
@@ -16,6 +16,7 @@
 
 import * as yup from 'yup';
 import { DescriptorEnvelope } from '../ingestion';
+import { DatabaseLocationUpdateLogEvent } from '../database';
```

Comment:
This file should not import from database. The types here should be ones that are suitable for direct consumption by clients of the service, and that would not typically include raw DB data.

Reply:
- [x] remove type import